processes-services/src/main/java/org/exoplatform/processes/service/ProcessesServiceImpl.java

```diff
@@ -211,7 +219,7 @@ public void start() {
         spaceService.updateSpace(existSpace);
       }
     } catch (Exception e) {
-      LOG.error("Error while initializing Processes app default space");
+      LOG.error("Error while initializing Processes app default space", e);
```

Comment:
Please change the message to someting like "Error while creating the default space"